src/pages/AssetDetailsPage.vue

```diff
@@ -59,13 +53,13 @@ onMounted(async () => {
 const transactionStatus = ref<ftm.Status>(null);
 
 // FIXME: this loads ALL transactions, not only related to the asset
-const transactionsTable = useTable(transactionModel.fetchList);
+const transactionsTable = useTable(http.fetchTransactions);
```

Comment:
Let's remove this table from the asset page altogether. We cannot implement it in the nearest future.